qemu/tests/driver_load.py

```diff
@@ -79,11 +79,24 @@ def get_driver_id(cmd, pattern):
         session.close()
         return driver_id
 
+    def service_operate(cmd):
+        """
+        Stop/Start service
+        :param cmd: cmd to stop/start service
+        """
+        session = vm.wait_for_login()
+        status, output = session.cmd_status_output(cmd)
+        if status != 0:
+            raise exceptions.TestFail("Service operation failed: %s" % output)
+        session.close()
```

Comment:
as there are too many subtest, two many alive session will make the fd full, it's ok to remove it if session fd clean works well in framework